### PR TITLE
docs: fix PrevAndNext icons RTL

### PR DESCRIPTION
**Pull request by qmhc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix incorrect prev and next icons of footer in RTL |
| 🇨🇳 Chinese | 修复在 RTL 时页脚上/下一章图标错误的问题 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 23798ed</samp>

This pull request adds support for right-to-left languages in the documentation theme by passing an `rtl` prop to the `PrevAndNext` component and using appropriate icons for navigation. The changes affect the files `.dumi/theme/common/PrevAndNext.tsx` and `.dumi/theme/slots/Content/index.tsx`.

### <samp>🤖 Generated by Copilot at 23798ed</samp>

*  Add support for right-to-left languages in the documentation theme
  - Pass `rtl` prop to `PrevAndNext` component in `Content` component ([link](https://github.com/ant-design/ant-design/pull/42822/files?diff=unified&w=0#diff-1533ad458dddb5cacbf24062cc5d262ca684a57e2754bf4777f24f37411d5f82L306-R306))
  - Use conditional rendering to assign appropriate icons to `before` and `after` props of `useMenu` hook in `PrevAndNext` component ([link](https://github.com/ant-design/ant-design/pull/42822/files?diff=unified&w=0#diff-f11fa3a3e55666da1f421678270305039a61b104fba0d4cce4d18959b8d9cf42L97-R107))

### CODE REVIEW

File: .dumi/theme/common/PrevAndNext.tsx [hunk 1]
```diff
@@ -94,13 +94,16 @@ const flattenMenu = (menuItems: MenuProps['items']): MenuProps['items'] | null =
...
   const { styles } = useStyle();
 
-  const [menuItems, selectedKey] = useMenu({
-    before: <LeftOutlined className="footer-nav-icon-before" />,
-    after: <RightOutlined className="footer-nav-icon-after" />,
-  });
+  const beforeProps = { className: 'footer-nav-icon-before' };
+  const afterProps = { className: 'footer-nav-icon-after' };
+
+  const before = rtl ? <RightOutlined {...beforeProps} /> : <LeftOutlined {...beforeProps} />;
```
Comment (li-jia-nan) [inline .dumi/theme/common/PrevAndNext.tsx:103]:
```suggestion
  const before = rtl ? <LeftOutlined{...beforeProps} /> : <RightOutlined{...beforeProps} />;
```

写反了

File: .dumi/theme/common/PrevAndNext.tsx [hunk 2]
```diff
@@ -94,13 +94,16 @@ const flattenMenu = (menuItems: MenuProps['items']): MenuProps['items'] | null =
...
+  const after = rtl ? <LeftOutlined {...afterProps} /> : <RightOutlined {...afterProps} />;
```
Comment (li-jia-nan) [inline .dumi/theme/common/PrevAndNext.tsx:104]:
```suggestion
  const after = rtl ? <RightOutlined {...afterProps} /> : <LeftOutlined {...afterProps} />;
```
写反了

**Comment by li-jia-nan** [comment]
好像还是不对……

**Review by afc163 — APPROVED** [review]

**Comment by li-jia-nan** [comment]
大佬，这个先不能合，还有点问题

**Comment by kiner-tang** [comment]
> 大佬，这个先不能合，还有点问题

如果是 member 的 PR 是不是其他 member +1 就好，啥时候合并由作者决定，这样可以避免这个问题

**Comment by li-jia-nan** [comment]
> > 大佬，这个先不能合，还有点问题
> 
> 如果是 member 的 PR 是不是其他 member +1 就好，啥时候合并由作者决定，这样可以避免这个问题

不用这么麻烦，有问题直接留个言就行了，该合就合，偏右大佬可能没看到我的评论

**Comment by afc163** [comment]
确实没看到。。

**Comment by afc163** [comment]
可以给不通过的 review

**Comment by yoyo837** [comment]
> > 大佬，这个先不能合，还有点问题
> 
> 如果是 member 的 PR 是不是其他 member +1 就好，啥时候合并由作者决定，这样可以避免这个问题

可以挂[WIP]

**Comment by kiner-tang** [comment]
> > > 大佬，这个先不能合，还有点问题
> > 
> > 
> > 如果是 member 的 PR 是不是其他 member +1 就好，啥时候合并由作者决定，这样可以避免这个问题
> 
> 可以挂[WIP]

有些时候可能是刚发现的问题还没来得及回复啥的就被合并了（我之前试过好多次，都是刚发现问题，一回来发现已经被合了），个人还是觉得，如果是 member 的话，是否合并的决定权还是交给作者好一点，其他 reviewer 如果没问题了，就 Approve 即可。

**Comment by li-jia-nan** [comment]
作者不一定有合并的权限呀……不是每个人都是member

**Comment by yoyo837** [comment]
不用太严格吧, 招呼一声+WIP+draft等就够了, 就算认真review过的也经常会有看不到的问题, 重发PR跟进呗.

**Comment by kiner-tang** [comment]
> 不用太严格吧, 招呼一声+WIP+draft等就够了, 就算认真review过的也经常会有看不到的问题, 重发PR跟进呗.

感觉可以

**Pull request merged**